src/java.naming/share/classes/com/sun/jndi/ldap/VersionHelper.java

```diff
@@ -109,14 +107,8 @@ Class<?> loadClass(String className) throws ClassNotFoundException {
         return Class.forName(className, true, getContextClassLoader());
     }
 
-    @SuppressWarnings("removal")
     Thread createThread(Runnable r) {
-        AccessControlContext acc = AccessController.getContext();
-        // 4290486: doPrivileged is needed to create a thread in
-        // an environment that restricts "modifyThreadGroup".
-        PrivilegedAction<Thread> act =
-                () -> SharedSecrets.getJavaLangAccess().newThreadWithAcc(r, acc);
-        return AccessController.doPrivileged(act);
+        return new Thread(r);
```

Comment:
Can the refactoring go back to the callers of createThread() to just create threads themselves.
And remove this trivial method.

Reply:
Yes, but in the PR that does the cleanup of java.naming module.